ios/TextilePhotos.xcodeproj/project.pbxproj

```diff
@@ -1205,6 +1205,9 @@
 							com.apple.BackgroundModes = {
 								enabled = true;
 							};
+							com.apple.Push = {
+								enabled = 0;
+							};
```

Comment:
Check that out. Automatically  updated by removing the methods from AppDelegate

Reply:
wooooow